### PR TITLE
ref(ui): Add missing types to project sourcemaps

### PR DESCRIPTION
**Pull request by scttcper**
found a few missing types
part of https://github.com/getsentry/frontend-tsc/issues/79

### CODE REVIEW

File: static/app/views/settings/projectSourceMaps/sourceMapsDetails.tsx [hunk 1]
```diff
@@ -255,7 +255,9 @@ export function SourceMapsDetails({params, location, router, project}: Props) {
                   key={data.id}
                   size={data.fileSize}
                   name={data.filePath}
-                  type={debugIdBundleTypeLabels[data.fileType]}
+                  type={
+                    debugIdBundleTypeLabels[data.fileType as DebugIdBundleArtifactType]
```
Comment (scttcper) [inline static/app/views/settings/projectSourceMaps/sourceMapsDetails.tsx:259]: data.fileType is a number and debugIdBundleTypeLabels uses an enum as the keys

File: static/app/views/settings/projectSourceMaps/sourceMapsList.tsx [hunk 2]
```diff
@@ -69,7 +70,19 @@ function mergeReleaseAndDebugIdBundles(
   return [...debugIdUploads, ...releaseUploads] as SourceMapUpload[];
 }
 
-function useSourceMapUploads({organization, project, query, cursor}) {
+interface UseSourceMapUploadsProps {
+  cursor: string | undefined;
```
Comment (priscilawebdev) [inline static/app/views/settings/projectSourceMaps/sourceMapsList.tsx:74]: shouldn't this be optional?

File: static/app/views/settings/projectSourceMaps/sourceMapsList.tsx [hunk 3]
```diff
@@ -69,7 +70,19 @@ function mergeReleaseAndDebugIdBundles(
...
+  organization: Organization;
+  project: Project;
+  query: string | undefined;
```
Comment (priscilawebdev) [inline static/app/views/settings/projectSourceMaps/sourceMapsList.tsx:77]: shouldn't this be optional?

**Review by priscilawebdev — APPROVED** [review]
Good catch @scttcper 👏 thanks for adding types!

File: static/app/views/settings/projectSourceMaps/sourceMapsList.tsx (hunk 2 above)
Comment (scttcper) [reply static/app/views/settings/projectSourceMaps/sourceMapsList.tsx:74]: at the moment they're always provided. can be made optional if someone needs that later

**Pull request merged**